homeassistant/components/freebox/binary_sensor.py

```diff
@@ -33,21 +35,99 @@
 async def async_setup_entry(
     hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
 ) -> None:
-    """Set up the binary sensors."""
+    """Set up binary sensors."""
     router: FreeboxRouter = hass.data[DOMAIN][entry.unique_id]
 
     _LOGGER.debug("%s - %s - %s raid(s)", router.name, router.mac, len(router.raids))
 
-    binary_entities = [
+    binary_entities: list[BinarySensorEntity] = [
         FreeboxRaidDegradedSensor(router, raid, description)
         for raid in router.raids.values()
         for description in RAID_SENSORS
     ]
 
+    for node in router.home_devices.values():
+        if node["category"] == FreeboxHomeCategory.PIR:
+            binary_entities.append(FreeboxPirSensor(hass, router, node))
+        elif node["category"] == FreeboxHomeCategory.DWS:
+            binary_entities.append(FreeboxDwsSensor(hass, router, node))
+
+        for endpoint in node["show_endpoints"]:
+            if (
+                endpoint["name"] == "cover"
+                and endpoint["ep_type"] == "signal"
+                and endpoint.get("value") is not None
+            ):
+                binary_entities.append(FreeboxCoverSensor(hass, router, node))
+
     if binary_entities:
         async_add_entities(binary_entities, True)
 
 
+class FreeboxHomeBinarySensor(FreeboxHomeEntity, BinarySensorEntity):
+    """Representation of a Freebox binary sensor."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        router: FreeboxRouter,
+        node: dict[str, Any],
+        sub_node: dict[str, Any] | None = None,
+    ) -> None:
+        """Initialize a Freebox binary sensor."""
+        super().__init__(hass, router, node, sub_node)
+        self._command_id = self.get_command_id(
+            node["type"]["endpoints"], "signal", "trigger"
+        )
+
+    async def async_update_signal(self):
+        """Update state."""
+        self._attr_is_on = await self.get_home_endpoint_value(self._command_id)
```

Comment:
May you test this ?

Or maybe we should reverse the value with `not` ?
I did not understand the previous `None` check, why this check existed ?

Reply:
At startup _attr_is_on  was None, so _attr_is_on == detection was always false and _attr_is_on was never updated

Reverse the value with not -> done 
Init value -> done 
Tests OK

Reply:
With that the init value is True, but should not as we don't know the state --> unknown

I don't understand why we have to reverse the value !
But if it should ... will see users issues !
You seem to say when movement => no movement